QGIS 3.8.0 fails to build with Qt 5.13.0 (incomplete type ‘QList<QgsInterpolator::LayerData>’)

During the build of QGIS 3.8.0 with Qt 5.13.0 on Arch Linux, I get the following error:
```
[ 30%] Building CXX object src/analysis/CMakeFiles/qgis_analysis.dir/interpolation/qgsinterpolator.cpp.o
In file included from /build/qgis/src/qgis-3.8.0/src/analysis/interpolation/qgsidwinterpolator.h:21,
                 from /build/qgis/src/qgis-3.8.0/src/analysis/interpolation/qgsidwinterpolator.cpp:18:
/build/qgis/src/qgis-3.8.0/src/analysis/interpolation/qgsinterpolator.h:142:22: error: field ‘mLayerData’ has incomplete type ‘QList<QgsInterpolator::LayerData>’
  142 |     QList<LayerData> mLayerData;
      |                      ^~~~~~~~~~
In file included from /usr/include/qt/QtCore/qglobal.h:1218,
                 from /usr/include/qt/QtCore/qalgorithms.h:43,
                 from /usr/include/qt/QtCore/qvector.h:43,
                 from /usr/include/qt/QtCore/QVector:1,
                 from /build/qgis/src/qgis-3.8.0/src/analysis/interpolation/qgsinterpolator.h:21,
                 from /build/qgis/src/qgis-3.8.0/src/analysis/interpolation/qgsidwinterpolator.h:21,
                 from /build/qgis/src/qgis-3.8.0/src/analysis/interpolation/qgsidwinterpolator.cpp:18:
/usr/include/qt/QtCore/qtypeinfo.h:212:1: note: declaration of ‘class QList<QgsInterpolator::LayerData>’
  212 | Q_DECLARE_MOVABLE_CONTAINER(QList);
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /build/qgis/src/qgis-3.8.0/src/analysis/interpolation/qgsidwinterpolator.h:21,
                 from /build/qgis/src/qgis-3.8.0/src/analysis/interpolation/qgsidwinterpolator.cpp:18:
/build/qgis/src/qgis-3.8.0/src/analysis/interpolation/qgsinterpolator.h:120:40: error: return type ‘class QList<QgsInterpolator::LayerData>’ is incomplete
  120 |     QList<LayerData> layerData() const { return mLayerData; } SIP_SKIP
      |                                        ^
```

--- a/src/analysis/interpolation/qgsinterpolator.h
+++ b/src/analysis/interpolation/qgsinterpolator.h
@@ -19,6 +19,7 @@
 #define QGSINTERPOLATOR_H
 
 #include <QVector>
+#include <QList>
 #include "qgis_sip.h"
 #include "qgis_analysis.h"
 

